src/main/java/seedu/address/model/person/Name.java

```diff
@@ -50,8 +53,46 @@ public boolean containsKeyword(String test) {
         if (test.equals("")) {
             return false;
         } else {
-            return this.fullName.toLowerCase().contains(test);
+            String[] keywords = test.split(" ");
+            String[] names = this.fullName.split(" ");
+            List<String> nameList = Arrays.asList(this.fullName.split(" "));
```

Comment:
May I know why `this.fullName.split(" ")` is used? Why not use `names`?

Reply:
You are right I didn't notice that